src/main/java/mods/railcraft/common/util/charge/ChargeNetwork.java

```diff
@@ -662,7 +663,7 @@ public void zap(Entity entity, Charge.DamageOrigin origin, float damage) {
                 }
                 if (remainingDamage > 0.1 && entity.attackEntityFrom(origin == Charge.DamageOrigin.BLOCK ? RailcraftDamageSource.ELECTRIC : RailcraftDamageSource.TRACK_ELECTRIC, remainingDamage)) {
                     removeCharge(chargeCost);
-                    Charge.effects().zapEffectDeath(entity.world, entity);
+                    HostEffects.INSTANCE.zapEffectDeath(entity.world, entity);
```

Comment:
This one needs to remain unchanged, if only so we know that API is working.

Reply:
Done. However, api needs another merge as you merged that pr prematurely.